sdks/java/core/src/test/java/org/apache/beam/sdk/testing/TestPipelineTest.java

```diff
@@ -112,23 +112,7 @@ public void testCreationOfPipelineOptionsFromReallyVerboselyNamedTestCase() thro
     public void testToString() {
       assertEquals(
           "TestPipeline#TestPipelineTest$TestPipelineCreationTest-testToString",
-          TestPipeline.create().toString());
-    }
-
-    @Test
-    public void testToStringNestedMethod() {
-      TestPipeline p = nestedMethod();
-
-      assertEquals(
-          "TestPipeline#TestPipelineTest$TestPipelineCreationTest-testToStringNestedMethod",
-          p.toString());
-      assertEquals(
-          "TestPipelineTest$TestPipelineCreationTest-testToStringNestedMethod",
-          p.getOptions().as(ApplicationNameOptions.class).getAppName());
-    }
-
-    private TestPipeline nestedMethod() {
-      return TestPipeline.create();
+          pipeline.toString());
```

Comment:
Please refer to JIRA " [BEAM-2630] TestPipeline: construct job/app names based on Description in junit TestRule."